src/test/java/ut/org/catrobat/jira/timesheet/servlet/JsonImporterServletTest.java

```diff
@@ -110,6 +114,32 @@ public void testDoPostTimesheet() throws IOException, ServletException {
         importTimesheetAsJsonServlet.doPost(request, response);
     }
 
+    @Test
+    public void testParseJSONTimesheet() throws IOException {
+        ImportTimesheetAsJsonServlet importTimesheetAsJsonServlet = new ImportTimesheetAsJsonServlet(loginUriProvider,
+                webSudoManager, permissionService, configService, timesheetService, entryService, categoryService, teamService,
+                renderer);
+
+        File file = new File("src/test/java/ut/org/catrobat/jira/timesheet/servlet/exampleJSONs/example1.json");
+        Gson gson = new Gson();
+        JsonTimesheetAndEntries[] result = importTimesheetAsJsonServlet.parseJSONToEntries(file, response, gson);
+
+        Assert.notNull(result);
+    }
+
+    @Test
+    public void testParseJSONTimesheet2() throws IOException {
+        ImportTimesheetAsJsonServlet importTimesheetAsJsonServlet = new ImportTimesheetAsJsonServlet(loginUriProvider,
+                webSudoManager, permissionService, configService, timesheetService, entryService, categoryService, teamService,
+                renderer);
+
+        File file = new File("src/test/java/ut/org/catrobat/jira/timesheet/servlet/exampleJSONs/example2.json");
+        Gson gson = new Gson();
+        JsonTimesheetAndEntries[] result = importTimesheetAsJsonServlet.parseJSONToEntries(file, response, gson);
+
+        Assert.notNull(result);
+    }
```

Comment:
Please also verify that the number of parsed entries is large enough (e.g., larger than 1000 or similar).